components/editor/modules/meta/PaynotesForm.js

```diff
@@ -0,0 +1,196 @@
+import React from 'react'
+
+import { A, Checkbox, Label, RawHtml } from '@project-r/styleguide'
+import MdClose from 'react-icons/lib/md/close'
+import MdAdd from 'react-icons/lib/md/add'
+import MdInfoOutline from 'react-icons/lib/md/info-outline'
+import { css } from 'glamor'
+
+import withT from '../../../../lib/withT'
+import PaynoteForm from './PaynoteForm'
+import { DARK_MODE_KEY } from './DarkModeForm'
+
+const styles = {
+  title: css({
+    display: 'block',
+    marginBottom: 5
+  }),
+  subtitle: css({
+    padding: '5px 0',
+    fontWeight: 'bold'
+  }),
+  container: css({
+    backgroundColor: '#fff',
+    padding: '5px 10px 15px',
+    marginBottom: 15
+  }),
+  close: css({
+    float: 'right'
+  }),
+  add: css({
+    marginTop: 10,
+    fontSize: 14
+  }),
+  help: css({
+    margin: '0 0 10px'
+  }),
+  checkbox: css({
+    display: 'inline-block',
+    marginRight: 10
+  })
+}
+
+const PAYNOTE_KEY = 'paynotes'
+const TARGETS = ['hasActiveMembership', 'isEligibleForTrial']
+
+const DEFAULT_TARGET = {
+  hasActiveMembership: false,
+  isEligibleForTrial: true
+}
+
+const DEFAULT_PAYNOTE = {
+  title: '',
+  body: '',
+  cta: 'trialForm',
+  button: {
+    label: 'Ich bin am Bord',
```

Comment:
Let's leave all fields here empty. Especially secondary is annoying and this combinations of `button` and `trialForm` doesn't make sense anyhow.

Reply:
Yes my point was to have an example – but maybe placeholder is more suitable indeed